fdbserver/storageserver.actor.cpp

```diff
@@ -12354,8 +12369,9 @@ void StorageServerDisk::writeMutations(const VectorRef<MutationRef>& mutations,
 bool StorageServerDisk::makeVersionMutationsDurable(Version& prevStorageVersion,
                                                     Version newStorageVersion,
                                                     int64_t& bytesLeft,
-                                                    UnlimitedCommitBytes unlimitedCommitBytes) {
-	if (!unlimitedCommitBytes && bytesLeft <= 0)
+                                                    UnlimitedCommitBytes unlimitedCommitBytes,
+                                                    int64_t& clearRangesLeft) {
+	if ((!unlimitedCommitBytes && bytesLeft <= 0) || clearRangesLeft <= 0)
```

Comment:
May be better to check clearRangesLeft==INT_MAX and if yes, does not do the clearRangesLeft check

Reply:
Discussed offline. clearRange amount is relatively small compared with INT_MAX. Also, we can check clearRange amount in general with the clearRangesLeft counter. So, we may want to decrement clearRangesLeft  in general.